.github/workflows/translate.yml

```diff
@@ -27,32 +27,22 @@ jobs:
           submodules: recursive
           ref: "master"
 
-      - name: install-deps
-        run: sudo apt install -y qt6-l10n-tools
-
       - name: l10n-remove-old
         run: |
-          rm translations/*.ts
-          git checkout translations/client_en.ts
-
-      - name: l10n-read
-        run: /usr/lib/qt6/bin/lupdate src -no-obsolete -ts translations/client_en.ts
+          git checkout ownCloud/Resources/
```

Comment:
That doesn't do anything, the desktop code deletes everything and then resets our source file, the en translations.